### DISCUSSION TITLE
Alternative/Feature request - package.json viewer

### DISCUSSION THREAD

**Discussion by mkandan** [gradejs/gradejs#138] [Ideas]
I'm pretty new to web development, but I'm wondering if there is an alternative (or maybe this is on your roadmap), or maybe this is impossible - but is there a way to see what packages a webapp is using?

For example, I want to know if Notion uses a package for their timeline view, or if they made it custom internally. This would almost be like being able to see their project's package.json.

**Comment by spalt08** [comment]
@mkandan https://gradejs.com/scan/notion.so
